rules.d/50-udev-default.rules.in

```diff
@@ -12,6 +12,7 @@ SUBSYSTEM=="rtc", KERNEL=="rtc0", SYMLINK+="rtc", OPTIONS+="link_priority=-100"
 
 SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", IMPORT{builtin}="usb_id", IMPORT{builtin}="hwdb --subsystem=usb"
 ENV{MODALIAS}!="", IMPORT{builtin}="hwdb --subsystem=$env{SUBSYSTEM}"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", IMPORT{builtin}="hwdb 'usb_device:v$attr{idVendor}p$attr{idProduct}:name:$attr{product}:'"
```

Comment:
why not reuse the same prefix? and just generate a shorter modalias? (i.e. one that lacks the interface bits?)

Reply:
I thought it would match both the interfaces, which have those modaliases, and the device itself, which would have meant that the interfaces would get tagged with uaccess too. But it doesn't really matter if both get tagged...

Reply:
maybe tighten the udev rule that applies uaccess, to do do so only if `ENV{DEVTYPE}=="usb_device"`. i.e. so that ID_SIGNAL_ANALYZER can be used liberally, but the uaccess tag is then only applied to to the real usb devices where we want that.

Reply:
I don't understand how else it would be applied though

Reply:
Like, I don't understand to what somebody would attach the `ID_SIGNAL_ANALYZER` property and not want it to change the access permissions if that was needed. Which then would mean that I need to change the explanation at the top of the hwdb file to note that the permission changes only apply to USB devices.

What do you reckon?